docs/guide/siteaccess.md

```diff
@@ -32,7 +32,7 @@ Typical uses of a SiteAccess are:
 The back-office UI of eZ Platform is housed in a predefined `admin` SiteAccess in `admin_group`.
 
 If you have a multisite setup with a separate back-office interface for each site,
-you need to create your own admin SiteAccesses for each site individually and add them to this group.
+you need to create your own admin SiteAccesses and add them to this group. In cases where the sites are on separate databases they'll need own repository config _(incl own storage and search connection)_, var dir, cache pool, as well as separate varnish/fastly cache in case that is in use for each site individually.
```

Comment:
I'm afraid that we will need an example of this configuration.

Reply:
Well, examples are available in the repository doc. what I can add here are some links to the related configuration.